scripts/source-maintenance.sh

```diff
@@ -12,15 +12,6 @@
 # on Squid
 #
 
-## Source Code Format Enforcement
-#
-# A checker to recursively reformat all source files: .h .c .cc .cci
-# using a custom astyle formatter and to use MD5 to validate that
-# the formatter has not altered the code syntax.
```

Comment:
I removed this stale and partially duplicated comment because we no longer depend on MD5. I can leave it intact if you prefer. IIRC, another pending PR will replace it anyway.